sockeye/layers.py

```diff
@@ -423,19 +445,17 @@ def __init__(self,
 
     def __call__(self,
                  queries: mx.sym.Symbol,
-                 queries_max_length: int,
                  memory: mx.sym.Symbol,
-                 memory_lengths: mx.sym.Symbol,
-                 memory_max_length: int) -> mx.sym.Symbol:
+                 bias: mx.sym.Symbol) -> mx.sym.Symbol:
         """
+        Computes multi-head attention for queries given a memory tensor.
+        Does not use an auto-regressive bias.
         Returns a symbol of shape (batch, max_length, output_depth).
 
-        :param queries: Symbol of shape (batch, queries_max_length, input_depth). TODO?
-        :param queries_max_length: Size of queries time dimension.
+        :param queries: Symbol of shape (batch, query_max_length, input_depth).
         :param memory: Symbol of shape (batch, memory_max_length, input_depth).
-        :param memory_lengths: Symbol of shape (batch, 1).
-        :param memory_max_length: Size of memory time dimension.
-        :return: Symbol of shape (batch, queries_max_length, output_depth).
+        :param bias: 3d bias. TODO Optional
```

Comment:
what is this "TODO Optional"?

Reply:
A TODO I apparently overlooked :) will fix